server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -5,18 +5,41 @@ const logger = require('@pubsweet/logger')
 
 const apiRoot = config.get('server.api.url')
 
-// request data from orcid API
-const request = (endpoint, query = {}) =>
-  superagent.get(apiRoot + endpoint).query(query)
-
-const convertPerson = apiPerson => ({
-  id: apiPerson.id,
-  name: apiPerson.name.preferred,
-  aff: apiPerson.affiliations && apiPerson.affiliations[0].name[0],
-  subjectAreas: (apiPerson.research.expertises || [])
-    .map(e => e.name)
-    .concat(apiPerson.research.focuses || []),
-})
+const request = (endpoint, query = {}) => {
+  const req = superagent.get(apiRoot + endpoint)
+
+  // only had the header if its defined in config
+  const secret = config.get('server.api.secret')
+  if (secret) {
+    req.header.Authorization = secret
+  }
+  return req.query(query)
+}
+
+const convertPerson = apiPerson => {
```

Comment:
Consider using default parameters instead of null checks going forward. This would need a separate PR to do this consistently over the codebase.

Reply:
See #1083